### ISSUE TITLE
No place to describe additional details

### ISSUE THREAD

**Issue by TheAdventurer64** [organicmaps/organicmaps#2772]
On the editor, there's a section asking you to describe "any additional details that cannot be edited in Organic Maps." However, there is no space to put this.![image](https://user-images.githubusercontent.com/50059322/174456612-082d4a0e-3493-47ac-9309-01aaac149e63.jpeg)

**Comment by vng** [comment]
You are right, the description and UI is confusing. It assumes that user presses "Place does not exist" button and writes a description there. The app in this case creates note in OSM.

**Comment by TheAdventurer64** [comment]
> You are right, the description and UI is confusing. It assumes that user presses "Place does not exist" button and writes a description there. The app in this case creates note in OSM.

In this case, a textbox like the one shown below should appear on every POI where you can add additional details, instad of just for POIs in the leisure category:![image](https://user-images.githubusercontent.com/50059322/181468509-b639376a-07e4-422e-a692-f303bd012b82.jpeg)

**Comment by map-per** [comment]
#9236 will fix this issue

**Comment by TheAdventurer64** [comment]
Looks like this was fixed. Closing
